src/main/resources/data/forge/tags/blocks/ores.json

```diff
@@ -0,0 +1,10 @@
+{
+    "replace": false,
+    "values": [
+        "#forge:ores/sapphire",
+        "#forge:ores/ruby"
+    ],
+    "optionalk": [
```

Comment:
I think this is a typo? shouldn't it be "optional" instead?

Reply:
Oof. Yep, its a typo.
Should be fixed now.